arbitrator/caller-env/src/static_caller.rs

```diff
@@ -105,7 +100,9 @@ impl MemAccess for StaticMem {
 }
 
 impl ExecEnv for StaticExecEnv {
-    fn print_string(&mut self, _data: &[u8]) {} // TODO?
+    fn print_string(&mut self, _data: &[u8]) {
+        // To print, call the debug-only `console::log_txt` Host I/O
```

Comment:
print_string is not for stylus programs but for the replay.wasm. We probably should add an actual print here